### DISCUSSION TITLE
[Suggestion] Separate configs for different modded deepslate and raw ores

### DISCUSSION THREAD

**Discussion by RaidhoSketch** [Roadhog360/Et-Futurum-Requiem#447] [Improvements]
Currently, only one config disables all raw-ore content - including the compat blocks. (And **yes**, modcompat.cfg has an option for disabling those, but only as a master-switch.) We ask for a separate config for enabling just _some_ raw ores, as well as deepslate ones - this is mainly applicable when someone already has a custom ore generator (like, say, with CustomOreGen), or just wants to play without them.

**Comment by Roadhog360** [comment]
Something along these lines is currently being brainstormed, I'm just unsure the best way to do it at the moment.

**Comment by Roadhog360** [comment]
Implemented in [e56a038](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/e56a038e7beed779fa5319716877636965ebdc10).
